objects-and-methods/exercise-1/lib/bag.rb

```diff
@@ -0,0 +1,27 @@
+class Bag
+
+    def initialize
+        @candies = []
+    end
+
+    def empty?
+        @candies.length == 0
```

Comment:
```suggestion
       @candies.empty?
 ```
you can use `empty?` here as well, it is cleaner and faster

Reply:
ok